.gitignore

```diff
@@ -18,5 +18,6 @@
 #Don't ignore requirements.txt
 !requirements.txt
 
-#Don't ignore setup
+#Don't ignore select scripts
```

Comment:
Why not `!*.sh`?

Reply:
I'm not sure we want just any script to be able to be added.  We have `!*.py` in the `.gitignore` but it seems like it takes some more thought to develop a python script than a shell script.  By allowing any shell script maybe we allow something to be committed by mistake.  Given that we have just two shell scripts at the moment I don't see the need to have `!*.sh`.

However, I'm not opposed to adding it.

Reply:
OK, though from my perspective, if people aren't using `git commit -a` we're fine :)